ui/src/app.vue

```diff
@@ -104,6 +104,11 @@
 
                 <!--detail header-->
                 <b-row>
+                    <b-col>
+                        <div  v-if="app.stats && app.stats.monthlyCounts && app.stats.monthlyCounts.length">
+                            <ExportablePlotly v-if="appStats" :data="appStats" :layout="statsLayout"/>
```

Comment:
@bhatiadheeraj Like you were saying, I think plotly graph might be too heavy for this? You can use visjs also which is a bit lighter than plotly graph. You can also just render a simple graph manually with a few simple <div> or SVG drawing.